docs/opentracing.asciidoc

```diff
@@ -5,8 +5,8 @@ The Elastic APM Go agent provides an implementation of the https://opentracing.i
 building on top of the core Elastic APM API.
 
 Spans created through the OpenTracing API will be translated to Elastic APM transactions or spans.
-Root spans, and spans created with a remote span context, will be translated to Elastic APM
-transactions; all others will be created as Elastic APM spans.
+Root spans and spans created with a remote span context, are translated to Elastic APM
```

Comment:
It's awkward to change from future tense here, but leave it elsewhere in this description.

Reply:
```suggestion
Root spans, and spans created with a remote span context, will be translated to Elastic APM
```